lib-multisrc/zorotheme/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 1
+baseVersionCode = 2
```

Comment:
This should only be bumped on a theme change, (and you shouldn't bump both this and `overrideVersionCode`, only bump `overrideVersionCode`.